src/models/namenode.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+from typing import List
+
+class Block(BaseModel):
+    Master: str
+    Slaves: List[str]
+
+class MetaData(BaseModel):
```

Comment:
between the classes there has to be two spaces. run flak8 or black to format the code.